Header header
int32 defectsType  # 0, 1, 2, 3